address;source_location;insn;indent
0x157fd0;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:86;PUSH	%RBX;
0x157fd1;:0;MOV	%RDI,%RBX;
0x157fd4;:0;SUB	$0x10,%RSP;
0x157fd8;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:75;MOV	-0x10(%RDI),%RCX;
0x157fdc;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:74;MOV	-0x8(%RDI),%RDX;
0x157fe0;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:86;MOV	%FS:0x28,%RAX;
0x157fe9;:0;MOV	%RAX,0x8(%RSP);
0x157fee;:0;XOR	%EAX,%EAX;
0x157ff0;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:77;MOV	0x8(%RCX),%RAX;
0x157ff4;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:74;AND	$-0x4,%RDX;
0x157ff8;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:76;MOV	%RCX,(%RDX);
0x157ffb;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:77;AND	$0x3,%EAX;
0x157ffe;:0;OR	%RDX,%RAX;
0x158001;:0;MOV	%RAX,0x8(%RCX);
0x158005;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:78;MOVQ	$0,-0x10(%RDI);
0x15800d;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_object.h:79;ANDQ	$0x1,-0x8(%RDI);
0x158012;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:76;MOV	0x10(%RDI),%RDI;
0x158016;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:76;TEST	%RDI,%RDI;
0x158019;:0;JNE	1581ac <BaseException_dealloc+0x1dc>;
0x15801f;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:77;MOV	0x18(%RBX),%RDI;
0x158023;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:77;TEST	%RDI,%RDI;
0x158026;:0;JE	15803c <BaseException_dealloc+0x6c>;
0x158028;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:77;MOVQ	$0,0x18(%RBX);
0x158030;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RDI);
0x158033;:0;JNE	15803c <BaseException_dealloc+0x6c>;
0x158035;/usr/local/src/conda/python-3.9.19/Objects/object.c:2205;MOV	0x8(%RDI),%RAX;
0x158039;/usr/local/src/conda/python-3.9.19/Objects/object.c:2209;CALLQ	0x30(%RAX);
0x15803c;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:78;MOV	0x20(%RBX),%RDI;
0x158040;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:78;TEST	%RDI,%RDI;
0x158043;:0;JE	158056 <BaseException_dealloc+0x86>;
0x158045;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:78;MOVQ	$0,0x20(%RBX);
0x15804d;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RDI);
0x158050;:0;JE	1581a2 <BaseException_dealloc+0x1d2>;
0x158056;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:79;MOV	0x30(%RBX),%RDI;
0x15805a;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:79;TEST	%RDI,%RDI;
0x15805d;:0;JNE	1582fe <BaseException_dealloc+0x32e>;
0x158063;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:80;MOV	0x28(%RBX),%RDI;
0x158067;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:80;TEST	%RDI,%RDI;
0x15806a;:0;JNE	158158 <BaseException_dealloc+0x188>;
0x158070;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:89;MOV	0x8(%RBX),%RAX;
0x158074;:0;LEA	-0x27a6b(%RIP),%RDX;
0x15807b;:0;MOV	0x140(%RAX),%RAX;
0x158082;:0;CMP	%RDX,%RAX;
0x158085;:0;JNE	158353 <BaseException_dealloc+0x383>;
0x15808b;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:2351;MOV	-0x10(%RBX),%RAX;
0x15808f;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:2350;LEA	-0x10(%RBX),%RDI;
0x158093;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:2351;TEST	%RAX,%RAX;
0x158096;:0;JNE	1582da <BaseException_dealloc+0x30a>;
0x15809c;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_pystate.h:55;MOV	0x25de55(%RIP),%RAX;(1299) 
0x1580a3;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:2355;MOV	0x10(%RAX),%RDX;(1299) 
0x1580a7;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:2356;MOV	0x294(%RDX),%EAX;(1299) 
0x1580ad;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:2356;TEST	%EAX,%EAX;(1299) 
0x1580af;:0;JLE	1580b9 <BaseException_dealloc+0xe9>;(1299) 
0x1580b1;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:2357;DEC	%EAX;(1299) 
0x1580b3;:0;MOV	%EAX,0x294(%RDX);(1299) 
0x1580b9;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:709;MOV	0x2512e0(%RIP),%RAX;(1299) 
0x1580c0;:0;LEA	-0x2e297(%RIP),%RDX;(1299) 
0x1580c7;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:709;MOV	0x2512b2(%RIP),%RCX;(1299) 
0x1580ce;:0;CMP	%RDX,%RAX;(1299) 
0x1580d1;:0;JNE	1582b7 <BaseException_dealloc+0x2e7>;(1299) 
0x1580d7;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1865;MOV	%RDI,%RAX;(1299) 
0x1580da;:0;AND	$-0x1000,%RAX;(1299) 
0x1580e0;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1420;MOV	0x20(%RAX),%EDX;(1299) 
0x1580e3;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1422;CMP	0x27e3df(%RIP),%EDX;(1299) 
0x1580e9;:0;JAE	1581c5 <BaseException_dealloc+0x1f5>;(1299) 
0x1580ef;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1422;MOV	0x27e612(%RIP),%RSI;(1299) 
0x1580f6;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1422;LEA	(%RDX,%RDX,2),%RDX;(1299) 
0x1580fa;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1422;MOV	%RDI,%RCX;(1299) 
0x1580fd;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1422;SAL	$0x4,%RDX;(1299) 
0x158101;:0;MOV	(%RSI,%RDX,1),%RDX;(1299) 
0x158105;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1422;SUB	%RDX,%RCX;(1299) 
0x158108;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1422;CMP	$0x3ffff,%RCX;(1299) 
0x15810f;:0;JA	1581c5 <BaseException_dealloc+0x1f5>;(1299) 
0x158115;:0;TEST	%RDX,%RDX;(1299) 
0x158118;:0;JE	1581c5 <BaseException_dealloc+0x1f5>;(1299) 
0x15811e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1878;MOV	0x8(%RAX),%RCX;(1299) 
0x158122;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1879;MOV	%RCX,-0x10(%RBX);(1299) 
0x158126;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1881;MOV	(%RAX),%EBX;(1299) 
0x158128;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1880;MOV	%RDI,0x8(%RAX);(1299) 
0x15812c;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1881;LEA	-0x1(%RBX),%EDX;(1299) 
0x15812f;:0;MOV	%EDX,(%RAX);(1299) 
0x158131;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1883;TEST	%RCX,%RCX;(1299) 
0x158134;:0;JE	158171 <BaseException_dealloc+0x1a1>;(1299) 
0x158136;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1897;TEST	%EDX,%EDX;(1299) 
0x158138;:0;JE	1581f5 <BaseException_dealloc+0x225>;(1299) 
0x15813e;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:90;MOV	0x8(%RSP),%RAX;(1299) 
0x158143;:0;SUB	%FS:0x28,%RAX;(1299) 
0x15814c;:0;JNE	1582d4 <BaseException_dealloc+0x304>;(1299) 
0x158152;:0;ADD	$0x10,%RSP;
0x158156;:0;POP	%RBX;
0x158157;:0;RET;
0x158158;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:80;XOR	%EAX,%EAX;
0x15815a;:0;MOV	%RAX,0x28(%RBX);
0x15815e;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RDI);
0x158161;:0;JNE	158070 <BaseException_dealloc+0xa0>;
0x158167;/usr/local/src/conda/python-3.9.19/Include/object.h:430;CALL	1296a0 <_Py_Dealloc>;
0x15816c;/usr/local/src/conda/python-3.9.19/Include/object.h:430;JMP	158070 <BaseException_dealloc+0xa0>;
0x158171;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1679;MOV	0x24(%RAX),%ESI;(1299) 
0x158174;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1679;LEA	0x2512c5(%RIP),%RCX;(1299) 
0x15817b;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1679;LEA	(%RSI,%RSI,1),%EDX;(1299) 
0x15817e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1679;MOV	(%RCX,%RDX,8),%RDX;(1299) 
0x158182;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1680;MOV	0x18(%RDX),%RCX;(1299) 
0x158186;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1683;MOVQ	%RDX,%XMM0;(1299) 
0x15818b;:0;MOVQ	%RCX,%XMM1;(1299) 
0x158190;:0;PUNPCKLQDQ	%XMM1,%XMM0;(1299) 
0x158194;:0;MOVAPS	%XMM0,0x10(%RAX);(1299) 
0x158198;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1685;MOV	%RAX,0x18(%RDX);(1299) 
0x15819c;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1686;MOV	%RAX,0x10(%RCX);(1299) 
0x1581a0;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1686;JMP	15813e <BaseException_dealloc+0x16e>;(1299) 
0x1581a2;/usr/local/src/conda/python-3.9.19/Include/object.h:430;CALL	1296a0 <_Py_Dealloc>;
0x1581a7;/usr/local/src/conda/python-3.9.19/Include/object.h:430;JMP	158056 <BaseException_dealloc+0x86>;
0x1581ac;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:76;XOR	%EAX,%EAX;
0x1581ae;:0;MOV	%RAX,0x10(%RBX);
0x1581b2;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RDI);
0x1581b5;:0;JNE	15801f <BaseException_dealloc+0x4f>;
0x1581bb;/usr/local/src/conda/python-3.9.19/Include/object.h:430;CALL	1296a0 <_Py_Dealloc>;
0x1581c0;/usr/local/src/conda/python-3.9.19/Include/object.h:430;JMP	15801f <BaseException_dealloc+0x4f>;
0x1581c5;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:595;MOV	0x251214(%RIP),%RAX;(1299) 
0x1581cc;:0;LEA	-0x340b3(%RIP),%RDX;(1299) 
0x1581d3;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:595;MOV	0x2511e6(%RIP),%RCX;(1299) 
0x1581da;:0;CMP	%RDX,%RAX;(1299) 
0x1581dd;:0;JNE	1583d8 <BaseException_dealloc+0x408>;(1299) 
0x1581e3;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:127;CALLQ	0x226307(%RIP);(1299) 
0x1581e9;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1923;DECQ	0x27e2d0(%RIP);(1299) 
0x1581f0;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1923;JMP	15813e <BaseException_dealloc+0x16e>;(1299) 
0x1581f5;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1700;MOV	0x20(%RAX),%EBX;(1299) 
0x1581f8;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1692;MOV	0x10(%RAX),%RCX;(1299) 
0x1581fc;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1693;MOV	0x18(%RAX),%RDX;(1299) 
0x158200;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1700;IMUL	$0x30,%RBX,%RBX;(1299) 
0x158204;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1694;MOV	%RDX,0x18(%RCX);(1299) 
0x158208;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1695;MOV	%RCX,0x10(%RDX);(1299) 
0x15820c;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1700;ADD	%RSI,%RBX;(1299) 
0x15820f;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1701;MOV	0x18(%RBX),%RDX;(1299) 
0x158213;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1703;MOV	0x10(%RBX),%EDI;(1299) 
0x158216;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1701;MOV	%RDX,0x10(%RAX);(1299) 
0x15821a;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1708;LEA	0x27e2df(%RIP),%RDX;(1299) 
0x158221;:0;MOV	(%RDX,%RDI,8),%RCX;(1299) 
0x158225;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1702;MOV	%RAX,0x18(%RBX);(1299) 
0x158229;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1703;MOV	%RDI,%RAX;(1299) 
0x15822c;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1715;CMP	%RCX,%RBX;(1299) 
0x15822f;:0;JNE	15823e <BaseException_dealloc+0x26e>;(1299) 
0x158231;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1716;MOV	0x28(%RBX),%RSI;(1299) 
0x158235;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1717;TEST	%RSI,%RSI;(1299) 
0x158238;:0;JNE	1582ae <BaseException_dealloc+0x2de>;(1299) 
0x15823a;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1717;MOV	%RSI,(%RDX,%RDI,8);(1299) 
0x15823e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1719;INC	%EAX;(1299) 
0x158240;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1719;MOV	%EAX,0x10(%RBX);(1299) 
0x158243;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1738;CMP	0x14(%RBX),%EAX;(1299) 
0x158246;:0;JE	15837d <BaseException_dealloc+0x3ad>;(1299) 
0x15824c;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1780;CMP	$0x1,%EAX;(1299) 
0x15824f;:0;JE	158317 <BaseException_dealloc+0x347>;(1299) 
0x158255;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1807;CMPQ	$0,(%RDX,%RAX,8);(1299) 
0x15825a;:0;JNE	158260 <BaseException_dealloc+0x290>;(1299) 
0x15825c;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1808;MOV	%RBX,(%RDX,%RAX,8);(1299) 
0x158260;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1811;CMP	%RCX,%RBX;(1299) 
0x158263;:0;JE	15813e <BaseException_dealloc+0x16e>;(1299) 
0x158269;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1825;MOV	0x28(%RBX),%RAX;(1299) 
0x15826d;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1738;MOV	0x20(%RBX),%RDX;(1299) 
0x158271;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1825;TEST	%RAX,%RAX;(1299) 
0x158274;:0;JE	158371 <BaseException_dealloc+0x3a1>;(1299) 
0x15827a;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1828;MOV	%RDX,0x20(%RAX);(1299) 
0x15827e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1835;MOV	0x20(%RBX),%RDX;(1299) 
0x158282;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1838;MOVQ	%RCX,%XMM2;(1299) 
0x158287;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1835;MOV	%RAX,0x28(%RDX);(1299) 
0x15828b;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1838;MOV	0x20(%RCX),%RAX;(1299) 
0x15828f;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1838;MOVQ	%RAX,%XMM0;(1299) 
0x158294;:0;PUNPCKLQDQ	%XMM2,%XMM0;(1299) 
0x158298;:0;MOVUPS	%XMM0,0x20(%RBX);(1299) 
0x15829c;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1839;TEST	%RAX,%RAX;(1299) 
0x15829f;:0;JE	1582a5 <BaseException_dealloc+0x2d5>;(1299) 
0x1582a1;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1840;MOV	%RBX,0x28(%RAX);(1299) 
0x1582a5;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1842;MOV	%RBX,0x20(%RCX);(1299) 
0x1582a9;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1842;JMP	15813e <BaseException_dealloc+0x16e>;(1299) 
0x1582ae;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1717;CMP	0x10(%RSI),%EDI;(1299) 
0x1582b1;:0;JE	15823a <BaseException_dealloc+0x26a>;(1299) 
0x1582b3;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1717;XOR	%ESI,%ESI;(1299) 
0x1582b5;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1717;JMP	15823a <BaseException_dealloc+0x26a>;(1299) 
0x1582b7;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:709;MOV	0x8(%RSP),%RDX;(1299) 
0x1582bc;:0;SUB	%FS:0x28,%RDX;(1299) 
0x1582c5;:0;JNE	1582d4 <BaseException_dealloc+0x304>;(1299) 
0x1582c7;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:90;ADD	$0x10,%RSP;
0x1582cb;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:709;MOV	%RDI,%RSI;
0x1582ce;:0;MOV	%RCX,%RDI;
0x1582d1;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:90;POP	%RBX;
0x1582d2;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:709;JMP	%RAX;
0x1582d4;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:90;CALLQ	0x22648e(%RIP);(1299) 
0x1582da;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:260;MOV	0x8(%RAX),%RDX;(1299) 
0x1582de;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:256;MOV	-0x8(%RBX),%RCX;(1299) 
0x1582e2;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:260;AND	$0x3,%EDX;(1299) 
0x1582e5;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:256;AND	$-0x4,%RCX;(1299) 
0x1582e9;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:260;OR	%RCX,%RDX;(1299) 
0x1582ec;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:259;MOV	%RAX,(%RCX);(1299) 
0x1582ef;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:260;MOV	%RDX,0x8(%RAX);(1299) 
0x1582f3;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:262;XOR	%EAX,%EAX;(1299) 
0x1582f5;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:262;MOV	%RAX,-0x10(%RBX);(1299) 
0x1582f9;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:262;JMP	15809c <BaseException_dealloc+0xcc>;(1299) 
0x1582fe;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:79;XOR	%EAX,%EAX;
0x158300;:0;MOV	%RAX,0x30(%RBX);
0x158304;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RDI);
0x158307;:0;JNE	158063 <BaseException_dealloc+0x93>;
0x15830d;/usr/local/src/conda/python-3.9.19/Include/object.h:430;CALL	1296a0 <_Py_Dealloc>;
0x158312;/usr/local/src/conda/python-3.9.19/Include/object.h:430;JMP	158063 <BaseException_dealloc+0x93>;
0x158317;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1786;MOV	0x27e1c2(%RIP),%RAX;(1299) 
0x15831e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1787;XOR	%R10D,%R10D;(1299) 
0x158321;:0;MOV	%R10,0x28(%RBX);(1299) 
0x158325;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1786;MOV	%RAX,0x20(%RBX);(1299) 
0x158329;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1788;TEST	%RAX,%RAX;(1299) 
0x15832c;:0;JE	158332 <BaseException_dealloc+0x362>;(1299) 
0x15832e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1789;MOV	%RBX,0x28(%RAX);(1299) 
0x158332;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1792;CMPQ	$0,0x27e1ce(%RIP);(1299) 
0x15833a;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1790;MOV	%RBX,0x27e19f(%RIP);(1299) 
0x158341;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1792;JNE	15813e <BaseException_dealloc+0x16e>;(1299) 
0x158347;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1793;MOV	%RBX,0x27e1ba(%RIP);(1299) 
0x15834e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1793;JMP	15813e <BaseException_dealloc+0x16e>;(1299) 
0x158353;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:89;MOV	0x8(%RSP),%RDX;
0x158358;:0;SUB	%FS:0x28,%RDX;
0x158361;:0;JNE	1582d4 <BaseException_dealloc+0x304>;
0x158367;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:90;ADD	$0x10,%RSP;
0x15836b;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:89;MOV	%RBX,%RDI;
0x15836e;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:90;POP	%RBX;
0x15836f;/usr/local/src/conda/python-3.9.19/Objects/exceptions.c:89;JMP	%RAX;
0x158371;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1833;MOV	%RDX,0x27e168(%RIP);(1299) 
0x158378;:0;JMP	15827e <BaseException_dealloc+0x2ae>;(1299) 
0x15837d;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1738;MOV	0x20(%RBX),%RSI;(1299) 
0x158381;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1738;TEST	%RSI,%RSI;(1299) 
0x158384;:0;JE	15824c <BaseException_dealloc+0x27c>;(1299) 
0x15838a;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1749;MOV	0x28(%RBX),%RAX;(1299) 
0x15838e;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1749;TEST	%RAX,%RAX;(1299) 
0x158391;:0;JE	703f5 <BaseException_dealloc.cold>;(1299) 
0x158397;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1756;MOV	%RSI,0x20(%RAX);(1299) 
0x15839b;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1762;MOV	%RAX,0x28(%RSI);(1299) 
0x15839f;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1768;MOV	0x27e132(%RIP),%RAX;(1299) 
0x1583a6;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1772;MOV	$0x40000,%EDX;(1299) 
0x1583ab;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1769;MOV	%RBX,0x27e126(%RIP);(1299) 
0x1583b2;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1772;MOV	(%RBX),%RSI;(1299) 
0x1583b5;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1768;MOV	%RAX,0x20(%RBX);(1299) 
0x1583b9;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1772;MOV	0x251280(%RIP),%RDI;(1299) 
0x1583c0;:0;CALLQ	0x25128a(%RIP);(1299) 
0x1583c6;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1774;XOR	%R11D,%R11D;(1299) 
0x1583c9;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1775;DECQ	0x27e100(%RIP);(1299) 
0x1583d0;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1774;MOV	%R11,(%RBX);(1299) 
0x1583d3;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:1777;JMP	15813e <BaseException_dealloc+0x16e>;(1299) 
0x1583d8;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:595;MOV	%RDI,%RSI;(1299) 
0x1583db;:0;MOV	%RCX,%RDI;(1299) 
0x1583de;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:595;CALL	%RAX;(1299) 
0x1583e0;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:595;JMP	1581e9 <BaseException_dealloc+0x219>;(1299) 
0x1583e5;/usr/local/src/conda/python-3.9.19/Objects/obmalloc.c:595;NOPW	%CS:(%RAX,%RAX,1);
